fiftyone/core/frame_utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+Frame utilites.
+
+| Copyright 2017-2020, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+from collections import defaultdict
+
+import six
+
+
+def is_frame_number(value):
+    if isinstance(value, six.integer_types):
+        if value < 0:
+            raise FrameError("positive ints only")
```

Comment:
Strictly positive, right? User can't pass frame number == 0 because we treat frame 1 as the first frame of the video

Reply:
Right, done.